docs/IntegrationWithExistingApps.md

```diff
@@ -782,3 +803,13 @@ if (!foundHash) {
   display('platform', isMac ? 'objc' : 'android');
 }
 </script>
+
+## Creating a release build in Android Studio
+
+You can use Android Studio to create your release builds too! It’s as easy as creating release builds of your previously-existing native Android app. There’s just one additional step, which you’ll have to do before every release build. You need to execute the following to create a React Native bundle, which’ll be included with your native Android app:
+
+    $ react-native bundle --platform android --dev false --entry-file index.android.js --bundle-output android/com/your-company-name/app-package-name/src/main/assets/index.android.bundle --assets-dest android/com/your-company-name/app-package-name/src/main/res/
```

Comment:
The `react.gradle` from node_modules should work when run via the command line `./gradlew assembleRelease`.

However it seems to skip the task when run through Android studio, I think it's occurs when the gradle [daemon](https://docs.gradle.org/2.14.1/userguide/gradle_daemon.html) is enabled.